src/map/pc.cpp

```diff
@@ -3605,13 +3587,8 @@ static void pc_bonus_itembonus(std::vector<s_item_bonus> &bonus, uint16 id, int
 
 	struct s_item_bonus entry = {};
 
-	if (cap_rate && (val < -10000 || val > 10000)) {
-		ShowWarning("pc_bonus_itembonus: Item bonus val %d exceeds -10000~10000 range, capping.\n", val);
-		val = cap_value(val, -10000, 10000);
-	}
-
 	entry.id = id;
-	entry.val = val;
+	entry.val = cap_value(val, -10000, 10000);
```

Comment:
It seem cap_rate boolean is deleted , so it make bSkillCooldown can reduce more than 10000 msec.